alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/field/LabelValueSelectOption.java

```diff
@@ -29,15 +29,16 @@
     private String value;
     private String icon;
 
-    public LabelValueSelectOption(final String labelAndValue) {
+    public LabelValueSelectOption(String labelAndValue) {
         this(labelAndValue, labelAndValue);
     }
 
-    public LabelValueSelectOption(final String label, final String value) {
-        this(label, value, null);
+    public LabelValueSelectOption(String label, String value) {
+        this.label = label;
+        this.value = value;
     }
 
-    public LabelValueSelectOption(final String label, final String value, final String icon) {
+    public LabelValueSelectOption(String label, String value, String icon) {
```

Comment:
Can this be deleted?

Reply:
It seems like we committed entirely to no icons. So this can be deleteed